modules/charon-core/src/main/java/org/wso2/charon3/core/objects/Group.java

```diff
@@ -208,6 +208,69 @@ private ComplexAttribute setMemberCommon(String userId, String userName, String
         return  complexAttribute;
     }
 
+    /**
+     * Set a member to the group, where member will have three default attributes such as name, value and $ref.
+     *
+     * @param user
+     * @throws BadRequestException
+     * @throws CharonException
+     */
+    public void setMember(User user) throws BadRequestException, CharonException {
+
+        if (this.isAttributeExist(SCIMConstants.GroupSchemaConstants.MEMBERS)) {
+            MultiValuedAttribute members = (MultiValuedAttribute) this.attributeList
+                    .get(SCIMConstants.GroupSchemaConstants.MEMBERS);
+            ComplexAttribute complexAttribute = setMemberCommon(user);
+            members.setAttributeValue(complexAttribute);
+        } else {
+            MultiValuedAttribute members = new MultiValuedAttribute(SCIMConstants.GroupSchemaConstants.MEMBERS);
+            DefaultAttributeFactory.createAttribute(SCIMSchemaDefinitions.SCIMGroupSchemaDefinition.MEMBERS, members);
+            ComplexAttribute complexAttribute = setMemberCommon(user);
+            members.setAttributeValue(complexAttribute);
+            this.setAttribute(members);
+        }
+    }
+
+    /**
+     * Create member attribute with three default attributes such as name, value and $ref.
+     *
+     * @param user
+     * @return
```

Comment:
Add a description to @return tag.

Reply:
Addressed with https://github.com/wso2/charon/pull/253